recipe/meta.yaml

```diff
@@ -35,11 +36,10 @@ requirements:
     - cmake
     - git
     - ninja
-    - macosx_deployment_target_osx-64 {{ MACOSX_DEPLOYMENT_TARGET }}  # [osx and x86_64]
+  host:
+    - libcublas-dev  # [(cuda_compiler_version or "").startswith("12")]
   run:
-    # llama.cpp *really* depends on the version it compiled against
-    - cudatoolkit {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
-    - macosx_deployment_target_osx-64 >={{ MACOSX_DEPLOYMENT_TARGET }}  # [osx and x86_64]
+    - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
```

Comment:
JFYI `{{ compiler('cuda') }}` [adds `cuda-version` automatically for CUDA 12]( https://github.com/conda-forge/cuda-nvcc-feedstock/blob/394f3817decd9316269b5a40e5076b8bcc94dc33/recipe/meta.yaml#L43-L45 )

With CUDA 11, `{{ compiler('cuda') }}` [adds `cudatoolkit`]( https://github.com/conda-forge/nvcc-feedstock/blob/c01599568a71b44a3026970d6b5666f9c7e0e8be/recipe/meta.yaml#L40-L46 ), which [`cuda-version` constrains]( https://github.com/conda-forge/cuda-version-feedstock/blob/286f34f62fe01701fdd5c38ed51da1341853b5b9/recipe/meta.yaml#L17 )

So this shouldn't be needed

Though please let me know if I'm missing something specific about this package

Reply:
@jakirkham, I got a few builds with CUDA 11.x where the cudatoolkit constraint was `>=11.X,<12`. This caused the tool to fail during runtime. I can get you the exception tomorrow if needed.

Thanks for checking!